docs/Deployment/Production-Checklist.md

```diff
@@ -8,15 +8,17 @@ menuOrder: 2
 This is a checklist for configuring and securing Cube.js for a production
 deployment.
 
-## Enable Production Mode
+## Disable Dev Mode
 
-When running Cube.js in production make sure `NODE_ENV` is set to `production` and `CUBEJS_DEV_MODE` is not set to `true`.
-Some platforms, such as Heroku, do it by default.
+When running Cube.js in production make sure dev mode is disabled. Running dev
+mode in production environment can lead to security vulnerabilities.
```

Comment:
I doubt that using "security vulnerabilities" here is appropriate because it really depends on the software using Cube.js and the nature of the data. IMO, it's quite misleading right now.

I think we can be more specific and add a relevant link here:

```suggestion
When running Cube.js in production environment, make sure the development mode is disabled. Running Cube.js in development
mode in a production environment allows accessing Cube.js API [without authorization](https://cube.dev/docs/security).
```